_src/src/utils.js

```diff
@@ -0,0 +1,9 @@
+export default {
+    getLanguage() {
+        if (navigator.language) {
+            return navigator.language;
+        } else {
+            return navigator.browserLanguage;
```

Comment:
This only works in some ancient browsers and even there, isn't it better to use `userLangauge`? (`browserLanguage` is supposed to show system's langage)

Reply:
`browserLanguage`  is show user select (IE can set in Internet Options dialog), but not system language, users may different between system and browser, I think follow browser is better